test/integration/replicaset/test_replicaset.py

```diff
@@ -37,6 +37,72 @@ def stop_application(tt_cmd, app_name, workdir, instances):
         assert not os.path.exists(os.path.join(workdir, run_path, app_name, inst, "tarantool.pid"))
 
 
+def start_cartridge_application(tt_cmd, tmpdir, cartridge_name):
```

Comment:
It would be nice to create session fixture that creates cartridge app and builds it. Then we reuse this application and stop+clear it after each test.